tuf/data/roles.go

```diff
@@ -68,6 +68,16 @@ func ValidRole(name string) bool {
 	return false
 }
 
+// IsBaseRole checks if a role name is valid base role
+func IsBaseRole(name string) bool {
```

Comment:
I would rather have a serialization method of a String to a Role Type that includes this logic.